lua/telescope/builtin/files.lua

```diff
@@ -201,11 +201,17 @@ files.file_browser = function(opts)
 
   opts.cwd = opts.cwd and vim.fn.expand(opts.cwd) or vim.loop.cwd()
 
-  local gen_new_finder = function(path)
-    opts.cwd = path
+  local gen_new_finder = function(new_opts)
+    opts.cwd = new_opts.path
+    opts.hidden = new_opts.hidden
```

Comment:
Question. First run i call `Telescope file_browser hidden=true` does hidden still gets applied becuase `new_opts` looks like this. `{ path = cwd }` so `opts.hidden` will be nil afterwards. I would do `opts.hidden = opts.hidden or new_opts.hidden` here.

Haven't tried it tho

Reply:
Btw everything in this PR looks really promising :) Great work.
I will try this PR in the tomorrow and give you more feedback. How much and what is still missing?

Reply:
I think everything is done, just need some more feedbacks :)